src/backend/src/main/java/com/linkedpipes/lpa/backend/controllers/UserController.java

```diff
@@ -174,4 +177,32 @@ public UserController(ApplicationContext context) {
         }
     }
 
+    @PostMapping("/api/user/application")
+    public ResponseEntity<UserProfile> addApplication(
+        @NotNull @RequestParam(value = "webId", required = true) String user,
+        @NotNull @RequestParam(value = "solidIri", required = true) String solidIri,
+        @NotNull @RequestParam(value = "executionIri", required = true) String executionIri)
+        throws LpAppsException {
+        try {
+            userService.addUserIfNotPresent(user);
```

Comment:
@eghuro any particular reason why we add user if not present? Does it make sense that the user wouldn't already be in database? Asking especially since then you're also catching UserNotFoundException..